### DISCUSSION TITLE
How to show more pre-defined colors resp find palette file?

### DISCUSSION THREAD

**Discussion by pstein** [tannerhelland/PhotoDemon#512] [General]
Currently in "Colors" sub pane the wheel and (possibly) the default history (of 14 colors) is shown (see attached snapshot).

I prefer to have more than these 14 colors at hand (but not a full wheel).
I want to have lets say pre-defined 64 colors at hand which I can pick from (a little bigger) boxes.

I guess this should be possible by selecting "palette" instead of "wheels + history" from the three dots "..." menu.

But I cannot select a (pre-defined) "palette" since Photodemons "palette" sub directory is empty.

When do I get a suitable palette file ?

![default palette v9 0](https://github.com/tannerhelland/PhotoDemon/assets/2775875/b805f7ee-da2e-4f50-8efb-dfa791f106b4)

**Comment by hi5** [comment]
1. You can create your own from a file, see File menu, Export, Pallete.
2. As you can see from the open Pallete option you can load Pallete from other graphics programs, so should you find one online (or have one already) you should be able to open it.
